src/main/java/com/cg/service/ManagerService.java

```diff
@@ -46,8 +62,90 @@ public String registerCustomer(int id)
 		return b.toString()+" "+customer.toString();
 		
 	}
-	public void generateBill()
+	public String generateBill(int id)
 	{
+
+		Customer customer=customerRepository.findById(id).get();
+		//Setting Bill Details
+		
+		Bill bill = new Bill();
+		int billId = count.incrementAndGet(); 
+		bill.setBillId(billId);
+		ZoneId zonedId = ZoneId.of( "Asia/Kolkata");
+		LocalDate date = LocalDate.now( zonedId );
+		bill.setDate(date);
+		double amount = customer.getParkingDuration()*30;
+		bill.setAmount(amount);
+	
+		//Generating Bill Receipt
+		try {
+			File billObj = new File("Bill.txt");
```

Comment:
As multiple customer are there to park. Therefore a common bill name "Bill.txt" is not sufficient. I think we must generate hash of customer object and use it as a bill name or some other way.